website/docs/reference/appsmith-architecture/README.md

```diff
@@ -0,0 +1,59 @@
+
+To manage the processes needed to function, Appsmith uses [Supervisor](http://supervisord.org/). This is a lightweight process manager that, when given a list of things to run, runs them, restarts them if they crash, and collects their logs for maintenance and debugging.
+
+When you run the Docker command to start the Appsmith container, it identifies which processes are required to start and run. This includes the backend server, the RTS, and the NGINX server -- as well as any additional processes that haven't been plugged in by the user externally, such as a custom MongoDB or Redis instance.
+
+For each of these processes, the container’s entry-point script (the one that Docker invokes when the container starts) generates a config file with all the details needed for supervisord. For example, the config file generated for the backend Java server includes the full set of commands for starting the process, as well as all necessary environment variables and settings. This same entry-point script also helps set up SSL certificates and install custom CA roots.
+
+Supervisord also comes with a command line tool that you can use to control these processes individually. For example, you can run `supervisorctl restart redis` if you want to restart the Redis process. Or, as another example, the `Save and Restart` button under `Admin Settings` on the Appsmith UI is powered the `supervisorctl` tool; clicking that button after you make changes with the UI prompts the backend server to execute `supervisorctl restart backend`, which restarts the backend process.
+
+### Docker
+
+Appsmith runs as a single Docker container, housing all five processes in one image via Supervisord.
+
+As described above in [process management](#process-management), this configuration provides flexibility in how you set up your Appsmith instance. Appsmith can take care of the entire batch of processes on its own, or you can choose to [drop in your own custom elements](/getting-started/setup/instance-configuration/custom-mongodb-redis) like MongoDB or Redis by modifying environment variables.
+
+### Kubernetes
+
+Kubernetes is an open source software used to manage and scale wide deployments of applications in containers; Appsmith Cloud runs on geographically distanced Kubernetes clusters via AWS. This deployment strategy allows the Appsmith Cloud platform to be scale and avoid downtime during outages, as well as conduct rolling updates with minimal disruptions to service.
+
+If you choose to [deploy Appsmith on Kubernetes](https://docs.appsmith.com/getting-started/setup/installation-guides/kubernetes), the only solution is to use our [official Helm chart](https://helm.appsmith.com). This Helm chart manages MongoDB and Redis as separate [Kubernetes pods](https://kubernetes.io/docs/concepts/workloads/pods/) so your Appsmith instances can scale horizontally. As before, this method also allows you the option to use external MongoDB and Redis instances that you manage on your own.
```

Comment:
Can we have relative paths linked instead of the URL for internal links?

Reply:
Oops!